src/utils/utils.js

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable import/prefer-default-export */
```

Comment:
I believe that once's you have two `export` this linting rule does not apply. You can remove it (unless I'm wrong).

Please name the module `timeRangeUtils.js`. It gives a better understanding of what we can find inside of the module. Naming is hard :)

Reply:
Indeed, it didn't apply anymore, thanks for pointing it out! Also, I couldn't agree more about naming.